Servers/utils/trainingRegistar.utils.ts

```
@@ -135,10 +139,11 @@ export const updateTrainingRegistarByIdQuery = async (
 
 export const deleteTrainingRegistarByIdQuery = async (
   id: number,
+  tenant: string,
   transaction: Transaction
 ): Promise<Boolean> => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)